### PR TITLE
[PAN-1981] Business logic for off-chain transfer submission

### PR DESCRIPTION
**Pull request by markuslevonyak**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Test(s)
- [x] Feature
- [ ] Bug Fix
- [x] Optimization
- [ ] Documentation Update

## Description

This PR adds the secondary nodes's business logic for getting and updating all required data for signing a cross-chain transfer and submitting the signature to the primary node. Additionally, all transfer-related tasks are scheduled only after the task's configured interval has passed.

### CODE REVIEW

File: pantos/validatornode/business/transfers.py [hunk 1]
```diff
@@ -908,11 +990,34 @@ def validate_transfer_task(self, internal_transfer_id: int,
...
         raise self.retry(countdown=retry_interval, exc=error)
     if not validation_completed:
-        retry_interval = config['tasks']['validate_transfer'][
-            'retry_interval_in_seconds']
+        retry_interval = _get_task_interval(self)
         raise self.retry(countdown=retry_interval)
     return True
+
+
+def _get_task_interval(task, after_error: bool = False) -> int:
```
Comment (danut13) [inline pantos/validatornode/business/transfers.py:1001]: Why does the celery task has no type?

Comment (markuslevonyak) [reply pantos/validatornode/business/transfers.py:1001]: Because Celery also has no typing, there's no advantage in adding one here.

**Review by danut13 — APPROVED** [review]

**Comment by jacekv** [comment]
I am not sure why the SonarCloud report currently does not work here - I thought that we had it fixed already. Anyway - I please have a look at the report and see if you could resolve some of those issues. Some should be trivial

**Review by jacekv — APPROVED** [review]

**Comment by markuslevonyak** [comment]
> I am not sure why the SonarCloud report currently does not work here - I thought that we had it fixed already. Anyway - I please have a look at the report and see if you could resolve some of those issues. Some should be trivial

All of them have been false positives and marked as such.

**Pull request merged**